### ISSUE TITLE
returning after previous LDAP auth gives a blank page

### ISSUE THREAD

**Issue by adq** [hashicorp/vault#9179]
**Describe the bug**
I logged in with LDAP: all fine.
I returned the next day, and all I see is a blank page from vault.

**To Reproduce**
1. Login with LDAP
2. Wait a bit
3. Blank page

**Expected behavior**
Not a blank page.

**Environment:**
* Vault Server Version (retrieve with `vault status`): 1.4.2
* Vault CLI Version (retrieve with `vault version`): 1.4.2
* Server Operating System/Architecture: Your docker image 1.4.2

Vault server configuration file(s):

```hcl
storage "file" {
  path = "/vault/file/"
}

listener "tcp" {
  address = "0.0.0.0:8200"
  tls_disable = 1
}

ui = true
default_lease_ttl = "168h"
max_lease_ttl = "720h"
```

**Additional context**
Looking in chrome debug, I only see some js files being requested over the network: no API calls.

In chrome console, I see:
```
vendor-29fe75b1145b75f4a79611c74b4f0a5a.js:2127 Error while processing route: vault.index Cannot read property 'includes' of null TypeError: Cannot read property 'includes' of null
    at https://myvaultserver/ui/assets/vault-41e3c883fe60cc5e9659c9c7bbbd2062.js:2308:15
    at Array.forEach (<anonymous>)
    at n.checkForRootToken (https://myvaultserver/ui/assets/vault-41e3c883fe60cc5e9659c9c7bbbd2062.js:2307:65)
    at n.init (https://myvaultserver/ui/assets/vault-41e3c883fe60cc5e9659c9c7bbbd2062.js:2277:242)
    at n [as init] (https://myvaultserver/ui/assets/vendor-29fe75b1145b75f4a79611c74b4f0a5a.js:2460:9)
    at m (https://myvaultserver/ui/assets/vendor-29fe75b1145b75f4a79611c74b4f0a5a.js:2369:218)
    at Function.e.create (https://myvaultserver/ui/assets/vendor-29fe75b1145b75f4a79611c74b4f0a5a.js:2378:8)
    at e.n.create (https://myvaultserver/ui/assets/vendor-29fe75b1145b75f4a79611c74b4f0a5a.js:881:18)
    at https://myvaultserver/ui/assets/vendor-29fe75b1145b75f4a79611c74b4f0a5a.js:855:76
    at l (https://myvaultserver/ui/assets/vendor-29fe75b1145b75f4a79611c74b4f0a5a.js:861:45)
```

Clicking on that shows: `n.policies.includes("root")` is the failing line.

If I look at the cached login details in my chrome browser, I see this:
```
{
   "backend" : {
      "description" : "LDAP authentication.",
      "displayNamePath" : "metadata.username",
      "formAttributes" : [
         "username",
         "password"
      ],
      "tokenPath" : "client_token",
      "type" : "ldap",
      "typeDisplay" : "LDAP"
   },
   "displayName" : "adq",
   "entity_id" : "MYENTITY",
   "policies" : null,
   "renewable" : true,
   "token" : "MYTOKEN",
   "tokenExpirationEpoch" : 1592165203112,
   "ttl" : 604800,
   "userRootNamespace" : ""
}
```
I notice policies is `null` - could this be the problem?

My user doesn't have any direct policies - its all done by groups.

This only seems to happen "after a while".. if I clear the token and login all is fine again, even if I close browser etc. I will check tomorrow and report back here. My browser's cached login details now has a full policies list: `["default", "netadmin", "testsecrets"]` rather than null.

**Comment by adq** [comment]
Hi, I've been unable to replicate this, however at least one other person did see it over here around the same time I did.

I wonder if its something caused when we set it up: we're using LDAP auth and had some faff to get LDAP groups working properly. Its now all ok though.

Obviously it can happen though, and the current failure mode is that the UI completely fails (is blank). I would suggest having a global try {} catch {} (or equivalent) so that it can be caught and at least give the user an option to clear + fix it.

**Comment by adq** [comment]
Hi, literally an hour since I submitted this, it just happened again. Unfortunately I was not watching my browser logs to see the start of the problem.

We have not changed the config on our server or anything in the last hour.

**Comment by chelshaw** [comment]
Hello and thank you for the thorough details on this bug report. Unfortunately I've been unable to replicate the behavior you describe, but we're going to go ahead and push a change that should catch a null `policies` value and at the very least not show a blank page. 

I notice you're currently on 1.4.2, are you looking for a minor release or are you and your team open to upgrading to the next major version (1.5.0) when it is released?

**Comment by chelshaw** [comment]
@adq I realized I forgot to tag you so you may have missed my message yesterday. Any thoughts on versions you're comfortable upgrading to?

**Comment by chelshaw** [comment]
The PR linked to this issue was merged and released with Vault 1.5.0, closing this out
